clang/lib/Sema/SemaType.cpp

```diff
@@ -10904,7 +10911,7 @@ class CountArgChecker : public TreeTransform<CountArgChecker> {
           VD->getType(), VK_LValue);
       return DRE;
     }
-
+  BOUNDS_SAFETY_CHECK_FAM_OR_ARROW_FOR_MEMBER:
```

Comment:
Oh an do we maybe want to extract this into a lambda or helper function instead of a goto label?